src/util/metamask.js

```diff
@@ -1,6 +1,33 @@
+import ENS from "ethjs-ens";
+
+
 export const assignEthereum = () => {
     if (typeof window.ethereum !== 'undefined') {
         return window.ethereum
     }
     return false
-}
\ No newline at end of file
+}
+
+const ens = new ENS({ provider: window.web3.currentProvider, network: 1 });
+
+export const convertENS = async (value) => {
+  if( value.includes('.eth') )
+  {
+    try {
+      const result = await ens.lookup(value);
+      return result
+    } catch (e) {
+      console.info(e.message);
+      return value
+    }
+  }
+  else {
```

Comment:
This else statement is redundant, as I mentioned above already, you should not be handling regular addresses in the ENS function. This should be for ENS domains only.